sdk/endpoints/online/managed/online-endpoints-managed-identity-uai.ipynb

```diff
@@ -0,0 +1,563 @@
+   "metadata": {
+    "name" : "give-access-permission-3"
+   },
+   "outputs": [],
+   "source": [
+    "workspace = ml_client.workspaces.get(workspace_name)\n",
+    "storage_account = workspace.storage_account\n",
+    "container_registry = workspace.container_registry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "give-access-permission-4"
+   },
+   "outputs": [],
+   "source": [
+    "role_name = \"AcrPull\"\n",
+    "scope = container_registry\n",
+    "\n",
+    "role_defs = auth_client.role_definitions.list(scope=scope)\n",
+    "role_def = next((r for r in role_defs if r.role_name == role_name))\n",
+    "\n",
+    "auth_client.role_assignments.create(\n",
+    "    scope=scope, \n",
+    "    role_assignment_name=uuid.uuid4(),\n",
+    "    parameters=RoleAssignmentProperties(\n",
+    "        role_definition_id = role_def.id, \n",
+    "        principal_id = uai_principal_id\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "give-access-permission-5"
+   },
+   "outputs": [],
+   "source": [
+    "role_name = \"Storage Blob Data Reader\"\n",
+    "scope = storage_account \n",
+    "\n",
+    "role_defs = auth_client.role_definitions.list(scope=scope)\n",
+    "role_def = next((r for r in role_defs if r.role_name == role_name))\n",
+    "\n",
+    "auth_client.role_assignments.create(\n",
+    "    scope=scope, \n",
+    "    role_assignment_name=uuid.uuid4(),\n",
+    "    parameters=RoleAssignmentProperties(\n",
+    "        role_definition_id = role_def.id, \n",
+    "        principal_id = uai_principal_id\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a deployment with your configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-deployment-1"
+   },
+   "outputs": [],
+   "source": [
+    "env_map = {\"STORAGE_ACCOUNT_NAME\" : storage_account_name, \n",
+    "\"STORAGE_CONTAINER_NAME\" : storage_container_name, \n",
+    "\"FILE_NAME\" : file_name,\n",
+    "\"UAI_CLIENT_ID\" : uai_client_id}\n",
+    "\n",
+    "for k, v in env_map.items():\n",
+    "    uai_deployment.environment_variables[k] = v\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "create-deployment-2"
+   },
+   "outputs": [],
+   "source": [
+    "uai_deployment = ml_client.online_deployments.begin_create_or_update(uai_deployment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Confirm your endpoint deployed successfully"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name" : "test-deployment-1"
+   },
+   "outputs": [],
+   "source": [
+    "sample_data = \"endpoints/online/model-1/sample-request.json\"\n",
```

Comment:
This path seems to be a relative path from `cli` directory.
Could you change it to the relative path from the .ipynb?

Reply:
Good catch, thank you.